server/BraveAlerterConfigurator.js

```diff
@@ -202,10 +230,14 @@ class BraveAlerterConfigurator {
     switch (fromAlertState) {
       case CHATBOT_STATE.STARTED:
       case CHATBOT_STATE.WAITING_FOR_REPLY:
-        returnMessage = t('incidentCategoryRequest', {
-          lng: language,
-          incidentCategories: this.createResponseStringFromIncidentCategories(incidentCategories, language),
-        })
+        if (toAlertState === CHATBOT_STATE.RESET) {
```

Comment:
Can you please add these `CHATBOT_STATE.RESET` states to the `getReturnMessageToOtherResponderPhoneNumbers` function below too in order to send the OtherResponderPhoneNumbers a message when the session is reset?

For some more context, you can see @tmackinnon  and me talking about it on Slack here: https://bravely.slack.com/archives/CAN15J43F/p1702943787701639?thread_ts=1702934030.289509&cid=CAN15J43F